inst/app/www/introduction.Rmd

```diff
@@ -4,13 +4,15 @@ output: html_document
 ---
 
 #### 1. Create your contributors table
-* Copy the contributors table template in your Google Drive _File -> Make a copy_
+* Follow the link in the app, copy the contributors table googlesheet template in your Google Drive _File -> Make a copy_
```

Comment:
This page is already on the how to use tenzing tab of the app. So I am not sure that I understand the follow the link in that app part.

Reply:
```suggestion
* Follow the link in the first tab of the app, copy the contributors table googlesheet template in your Google Drive _File -> Make a copy_
```

Reply:
that would be better ?